src/CalcViewModel/GraphingCalculator/GraphingSettingsViewModel.cpp

```diff
@@ -101,6 +101,7 @@ void GraphingSettingsViewModel::UpdateDisplayRange()
     }
 
     m_Graph->SetDisplayRanges(m_XMinValue, m_XMaxValue, m_YMinValue, m_YMaxValue);
+    m_Graph->RangeUpdatedBySettings = true;
```

Comment:
I am not a fan of the viewmodel keeping track of this state, can we set this to true/false in SetDisplayRanges instead and have it be a private member variable?